phoenix-core/src/main/java/org/apache/phoenix/compile/FromCompiler.java

```diff
@@ -196,7 +196,8 @@ public static ColumnResolver getResolverForCreation(final CreateTableStatement s
                     if (htable != null) Closeables.closeQuietly(htable);
                 }
                 tableNode = NamedTableNode.create(null, baseTable, statement.getColumnDefs());
-                return new SingleTableColumnResolver(connection, tableNode, e.getTimeStamp(), new HashMap<String, UDFParseNode>(1), false);
+                boolean isNamespaceMapped = SchemaUtil.isNamespaceMappingEnabled(statement.getTableType(), connection.getQueryServices().getProps());
```

Comment:
Originally the SingleTableColumnResolver constructor argument "isNamespaceMapped" is hard-coded as false, but we should set it from props.